vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -41,8 +41,8 @@ const (
 	devWaitTimeout   = 1 * time.Second
 	sleepBeforeMount = 1 * time.Second
 	watchPath        = "/dev/disk/by-path"
-	version          = "VMDK Volume Driver v0.3"
-	driverName       = "vmdk"
+	version          = "VSphere Volume Driver v0.3"
+	driverName       = "vsphere"
```

Comment:
I think we should keep, "vmdk" for backward compatibility. If possible warn user that "vmdk" driver type is deprecated and use "vSphere" instead.